samples/azure-quantum/variational-quantum-eigensolver/VQE-qiskit-hydrogen-ionq-sim.ipynb

```diff
@@ -0,0 +1,520 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "47574490",
+   "metadata": {},
+   "source": [
+    "# Simulate the ground state of a Hydrogen molecule using Variational Quantum Eigensolver (VQE) on the IonQ simulator\n",
```

Comment:
Because VQE can be quite expensive to run, this sample is meant for advanced users who already know what VQE is. It's not meant for absolute beginners - those folks should try out the getting started samples instead.

Reply:
Agreed, that's my point; giving readers that we don't expect to get use out of the sample an easy exit to learn more is important when a sample is more advanced like that.